docs/README.md

```diff
@@ -95,6 +95,13 @@ The block `product-price-savings` has two additional props:
 | `percentageStyle` | `locale` or `compact` | Set to `compact` if you want to remove the white space between the number and the percent sign. It uses pattern provided by the current locale as default. | `locale` |
 | `minimumPercentage` | `number` | Set the minimum value for the percentage value display. If not informed, it always appears. Example: `10`, savings under or equal 10% will not show up. | `0` |
 
+The block `product-installments-list` has two additional props:
```

Comment:
What if we had a section on README for the props info? I thought about that because another table was also added here in the other PR, I believe it would be better to consult the props info in a separate section and leave this one with the focus of being an example of how to add the block

Reply:
We usually have it a bit clearer, but the props are always shown under the `# Configuration` header.
Pretty much all of the docs you'll find in our open-source IO Apps were written with the help of @guerreirobeatriz, based on this template: https://github.com/vtex-apps/react-app-template#readme.

I don't have a clear opinion on creating a separate section just for props...  I guess I like the way we've been doing it because as soon as the block gets introduced, you already get to see what it can do. But I wouldn't mind having a separate section 😅 . @guerreirobeatriz likely has a better explanation for why it's structured like this

Reply:
Oh, good to know!